src/util/Parameters.h

```diff
@@ -156,6 +168,24 @@ struct MemorySizeFromString {
   }
 };
 
+template <typename T>
+struct StreamableConverter {
+  static_assert(!ad_utility::isSimilar<T, std::string>);
+
+  T operator()(const std::string& str) const {
+    std::istringstream is{str};
+    T result;
+    is >> result;
+    return result;
+  }
+
+  std::string operator()(const T& value) const {
+    std::ostringstream os;
+    os << value;
+    return std::move(os).str();
```

Comment:
As stated in the other file:
Maybe directly implement those functions as `fromString` and `toString`. You dcurrently parse from a string to a stream to a string and THEN do the actual parsing and need an uninitialized temporary etc.